k8s/services/imagepullsecrets/helm-0.3.cue

```diff
@@ -0,0 +1,77 @@
+package imagepullsecrets
+
+#KubeVersion: [=~"^0\\.3\\."]: minor: >=21
+#Values: [=~"^0\\.3\\."]: {
+	replicas: int | *1
+	istio: {
+		revision: ""
```

Comment:
this doesn't allow people to override this value maybe change it to `string | *""`